### ISSUE TITLE
Feature: Edit data in content view.

### ISSUE THREAD

**Issue by markmorris** [Alecaddd/sequeler#165]
Be able to edit the data when viewing the content of a table. Once that is done, i can completely replace MySQL Workbench and would be willing to pay twice for this software.

Keep up the good work!

**Comment by Alecaddd** [comment]
Posted twice? Closing this as well, duplicate of #67

**Comment by markmorris** [comment]
Sorry not sure why it posted twice, website was throwing out errors.
